pkg/diagnose/runner.go

```diff
@@ -440,12 +425,12 @@ func runStdOut(w io.Writer, diagCfg diagnosis.Config, run func(diagnosis.Config)
 			outputSuiteIfNeeded(w, ds.SuiteName, &suiteAlreadyReported)
 
 			outputNewLineIfNeeded(w, &lastDot)
-			outputDiagnosis(w, diagCfg, getDiagnosisResultForOutput(d.Result), c.total, d)
+			outputDiagnosis(w, diagCfg, d.Result.ToString(true), c.total, d)
 		}
 	}
 
 	outputNewLineIfNeeded(w, &lastDot)
-	c.summary(w)
+	c.summary(w, false)
```

Comment:
It seesm that the summary is only used in the non-json output. Do we still, need to add the boolean flag to signal the format type?

Reply:
just added in the next commit, I was waiting a way to add the connectivity result